mlir/test/mlir-miopen-driver/e2e/conv2d_host_validation_random.mlir

```diff
@@ -0,0 +1,63 @@
+// kcyx_nchw_nkhw
+// filter 3x3 f32
+// RUN: mlir-miopen-driver -pv -fil_layout=kcyx -in_layout=nchw -out_layout=nkhw  -batchsize=256 -in_channels=64 -out_channels=64 -in_h=56 -in_w=56 -fil_h=3 -fil_w=3 --dilation_h=1 --dilation_w=1 --padding_h=0 --padding_w=0 --conv_stride_h=1 --conv_stride_w=1 -p=false -rand 1 -c | mlir-rocm-runner  --shared-libs=%rocm_wrapper_library_dir/librocm-runtime-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CHECK_KCYX_NCHW_NKHW_4
+
```

Comment:
also, could we have sanity tests, and perhaps e2e tests to check setting different seeds

Reply:
@yiqian1 to be more clear, `sanity.mlir` shall be amended to ensure `-rand` flag is recognized.

Reply:
Isn't populate_random_data_seed.mlir sufficient to prove that?

Reply:
fair enough. that's sufficient.